<?xml version="1.0"?>
<launch>
    <include file="$(find rt1_a2_2023)/launch/sim_w1.launch" />
    
    <param name="des_pos_x" value= "5.0" />
    <param name="des_pos_y" value= "5.0" />
    
    <node pkg="rt1_a2_2023" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="rt1_a2_2023" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="rt1_a2_2023" type="bug_as.py" name="bug_action_service"  />
    <node pkg="rt1_a2_2023" type="DefineTargetService.py" name="DefineTargetService" output="screen" launch-prefix="xterm -hold -e"  />
    <node pkg="rt1_a2_2023" type="TargetDistanceService.py" name="TargetDistanceService" output="screen" launch-prefix="xterm -hold -e"  />
    <node pkg="rt1_a2_2023" type="TargetPositionService.py" name="TargetPositionService" output="screen" launch-prefix="xterm -hold -e" />
</launch>
